app/pages/browse.tsx

```diff
@@ -1,15 +1,30 @@
 import ArticleList from "app/core/components/ArticleList"
+import EnterDOI from "app/core/components/EnterDOI"
 import { Footer } from "app/core/components/Footer"
-import Navbar from "app/core/components/Navbar"
-import { BlitzPage } from "blitz"
+import { BlitzPage, useRouter, useSession } from "blitz"
 import { Suspense } from "react"
 
 const Browse = () => {
+  const session = useSession()
+  const router = useRouter()
+  const isAtRoot = router.pathname === "/"
+
   return (
-    <div className="flex flex-col min-h-screen">
-      <Navbar />
-      <main className="flex-grow flex flex-col items-center mb-4">
-        <ArticleList />
+    <div className="flex flex-col min-h-screen bg-white dark:bg-gray-darkest text-gray-darkest dark:text-white">
```

Comment:
We could add `overflow-y-clip` to prevent having the white space at the bottom. I'm also ok with the current implementation too! 👍 


<br class="Apple-interchange-newline">